Automattic/jetpack
# My Jetpack: Include featured image generation on Jetpack AI product interstitial

Fixes #37198.

## Proposed changes:

* Add "Generate featured images" as a feature on the product interstitial

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spin up a new site and connect it
* Go to the My Jetpack page and look for the AI product card

<img width="200" alt="Screenshot 2024-05-03 at 17 03 08" src="https://github.com/Automattic/jetpack/assets/6760046/90fceae2-2eba-4c56-93a6-11d439b84e76">

* Click the "Learn More" button
* You will be redirected to a product interstitial page with the features for the Jetpack AI product
* Confirm that "Generate featured images" is present on the list of features:

| Before | After |
| --- | --- |
| <img width="400" alt="Screenshot 2024-05-03 at 17 04 53" src="https://github.com/Automattic/jetpack/assets/6760046/5f997ab6-b1af-485d-9bc5-c620c893b98c"> | <img width="400" alt="Screenshot 2024-05-03 at 17 03 22" src="https://github.com/Automattic/jetpack/assets/6760046/92309c68-47b1-4918-8558-4bcaab0f2888"> |
